test/tests/slider-2.js

```diff
@@ -0,0 +1,583 @@
+
+ariaTest('role="slider" on div element', exampleFile, 'slider-role', async (t) => {
+  t.plan(1);
+  await assertAriaRoles(t, 'ex1', 'slider', '3', 'div');
+});
+
+ariaTest('"tabindex" set to "0" on sliders', exampleFile, 'tabindex', async (t) => {
+  t.plan(1);
+  await assertAttributeValues(t, ex.sliderSelector, 'tabindex', '0');
+});
+
+ariaTest('"aria-orientation" set on sliders', exampleFile, 'aria-orientation', async (t) => {
+  t.plan(3);
+  await assertAttributeValues(t, ex.tempSelector, 'aria-orientation', 'vertical');
+  await assertAttributeValues(t, ex.fanSelector, 'aria-orientation', 'horizontal');
+  await assertAttributeValues(t, ex.heatSelector, 'aria-orientation', 'horizontal');
+});
+
+ariaTest('"aria-valuemax" set on sliders', exampleFile, 'aria-valuemax', async (t) => {
+  t.plan(3);
+  await assertAttributeValues(t, ex.tempSelector, 'aria-valuemax', ex.tempMax);
+  await assertAttributeValues(t, ex.fanSelector, 'aria-valuemax', ex.fanMax);
+  await assertAttributeValues(t, ex.heatSelector, 'aria-valuemax', ex.heatMax);
+});
+
+ariaTest('"aria-valuemin" set on sliders', exampleFile, 'aria-valuemin', async (t) => {
+  t.plan(3);
+  await assertAttributeValues(t, ex.tempSelector, 'aria-valuemin', ex.tempMin);
+  await assertAttributeValues(t, ex.fanSelector, 'aria-valuemin', ex.fanMin);
+  await assertAttributeValues(t, ex.heatSelector, 'aria-valuemin', ex.heatMin);
+});
+
+ariaTest('"aria-valuenow" reflects slider value', exampleFile, 'aria-valuenow', async (t) => {
+  t.plan(3);
+  await assertAttributeValues(t, ex.tempSelector, 'aria-valuenow', ex.tempDefault);
+  await assertAttributeValues(t, ex.fanSelector, 'aria-valuenow', '0');
+  await assertAttributeValues(t, ex.heatSelector, 'aria-valuenow', '0');
+});
+
+ariaTest('"aria-valuetext" reflects slider value', exampleFile, 'aria-valuetext', async (t) => {
+  t.plan(2);
+  await assertAttributeValues(t, ex.fanSelector, 'aria-valuetext', ex.fanValues[0]);
+  await assertAttributeValues(t, ex.heatSelector, 'aria-valuetext', ex.heatValues[0]);
+});
+
+ariaTest('"aria-labelledby" set on sliders', exampleFile, 'aria-labelledby', async (t) => {
+  t.plan(1);
+  await assertAriaLabelledby(t, 'ex1', ex.sliderSelector);
+});
+
+
+// Keys
+
+ariaTest('Right arrow increases slider value by 1', exampleFile, 'key-right-arrow', async (t) => {
+  t.plan(8);
+
+  // Send 1 key to temp slider
+  const tempSlider = await t.context.session.findElement(By.css(ex.tempSelector));
+  await tempSlider.sendKeys(Key.ARROW_RIGHT);
+
+  let sliderVal = parseInt(ex.tempDefault) + 1;
+  t.is(
+    await tempSlider.getAttribute('aria-valuenow'),
+    sliderVal.toString(),
+    'After sending 1 arrow right key to the temp slider, "aria-valuenow": ' + sliderVal
+  );
+  t.is(
+    await t.context.session.findElement(By.css(ex.pageTempSelector)).getText(),
+    sliderVal.toString(),
+    'Temp display should match value of slider: ' + sliderVal
+  );
+
+  // Send more than 51 keys to temp slider
```

Comment:
Quibble: "Send 51 more keys to the temp slider" might make more sense here. It took me a minute to realize what this meant.